docs/root/configuration/http/http_filters/_include/local-rate-limit-with-descriptors.yaml

```diff
@@ -0,0 +1,97 @@
+                          - request_headers:
+                              header_name: x-envoy-downstream-service-cluster
+                              descriptor_key: client_cluster
+                          - request_headers:
+                              header_name: ":path"
+                              descriptor_key: path
+                      typed_per_filter_config:
+                        envoy.filters.http.local_ratelimit:
+                          "@type": type.googleapis.com/envoy.extensions.filters.http.local_ratelimit.v3.LocalRateLimit
+                          stat_prefix: test
+                          token_bucket:
+                            max_tokens: 1000
+                            tokens_per_fill: 1000
+                            fill_interval: 60s
+                          filter_enabled:
+                            runtime_key: test_enabled
+                            default_value:
+                              numerator: 100
+                              denominator: HUNDRED
+                          filter_enforced:
+                            runtime_key: test_enforced
+                            default_value:
+                              numerator: 100
+                              denominator: HUNDRED
+                          response_headers_to_add:
+                            - append: false
+                              header:
+                                key: x-test-rate-limit
+                                value: 'true'
+                          descriptors:
+                          - entries:
+                            - key: client_cluster
+                              value: foo
+                            - key: path
+                              value: /foo/bar
+                            token_bucket:
+                              max_tokens: 10
+                              tokens_per_fill: 10
+                              fill_interval: 60s
+                          - entries:
+                            - key: client_cluster
+                              value: foo
+                            - key: path
+                              value: /foo/bar2
+                            token_bucket:
+                              max_tokens: 100
+                              tokens_per_fill: 100
+                              fill_interval: 60s
+                    - match: { prefix: "/" }
+                      route: { cluster: default_service }
+  clusters:
+    - name: default_service
+      load_assignment:
+        cluster_name: default_service
+        endpoints:
+          - lb_endpoints:
+              - endpoint:
+                  address:
+                    socket_address:
+                      address: 127.0.0.1
+                      port_value: 8080
+    - name: service_protected_by_rate_limit
+      load_assignment:
+        cluster_name: service_protected_by_rate_limit
+        endpoints:
+          - lb_endpoints:
```

Comment:
not a blocker - but the mixed use of indenting styles for arrays can make config harder to grok

im going to clean these up at some point (pending pr is here #15726 ) 

generally least indent is preferred

Reply:
Done.